floris/simulation/turbine.py

```diff
@@ -424,6 +424,29 @@ def Cp(self):
         """
         return self._fCp(self.average_velocity)
 
+    # @property
+    def Cp_new(self, scaled_velocity):
```

Comment:
Ultimately, we should avoid names with '_new'. This will undoubtedly lead to issues in the future. We can add a deprication warning simply update the existing namespace.

Reply:
this should be fixed now